src/Config.ts

```diff
@@ -8,9 +8,17 @@ import * as configTypes from '../config/types.json';
 export interface ISidecarConfig {
 	HOST: string;
 	PORT: number;
-	LOG_MODE: string;
 	WS_URL: string;
 	CUSTOM_TYPES: Record<string, string> | undefined;
+	CONSOLE_LEVEL: string;
```

Comment:
Regarding `confmgr`: your App is split into a few modules.
I think it could be helpful to keep this structure here instead of flattening it.
That also allows passing a whole 'LoggerConfig' Object to the logger related code.

So in short (pseudo code):
```
export SideCarConfigExpress {
  HOST: string;
  PORT: number;
}

export SideCarConfigLogs {
  ...
}


export interface ISidecarConfig {
    EXPRESS: SideCarConfigExpress;
    LOGS: SideCarConfigLogs;
    ....
}
```